public/usage-examples/sprites/sprite_set_velocity/sprite_set_velocity-1-simple-oop.cs

```diff
@@ -0,0 +1,39 @@
+using SplashKitSDK;
+
+namespace velocity
+{
+    public class Program
+    {
+        public static void Main()
+        {
+            // Open a new window
```

Comment:
The comment here and on line 12, 21, and 35 do not exits in the other files. Either remove them from here or add them into existing files.